temporal-test-server/src/main/java/io/temporal/internal/testservice/TestWorkflowService.java

```diff
@@ -1465,6 +1488,7 @@ public String continueAsNew(
       String identity,
       ExecutionId continuedExecutionId,
       String firstExecutionRunId,
+      TestWorkflowMutableState previousExecutionState,
```

Comment:
Need the previous mutable state so we can copy all callbacks to the continued as new workflow (L1511-1514 below).